docs/source/torch.compiler_faq.rst

```diff
@@ -602,35 +602,84 @@ context. Consider the example
    Y = np.random.randn(1024, 64)
    with torch.device("cuda"):
        Z = numpy_fn(X, Y)
-
+   assert isinstance(Z, np.ndarray)
 
 In this example, ``numpy_fn`` will be executed in CUDA. For this to be
 possible, ``torch.compile`` automatically moves ``X`` and ``Y`` from CPU
 to CUDA, and then it moves the result ``Z`` from CUDA to CPU. If we are
 executing this function several times in the same program run, we may want
 to avoid all these rather expensive memory copies. To do so, we just need
-to tweak our ``numpy_fn`` so that it accepts cuda Tensors and returns tensors:
+to tweak our ``numpy_fn`` so that it accepts cuda Tensors and returns tensors.
+We can do so by using ``torch.compiler.wrap_numpy``:
```

Comment:
Thanks for the update, it looks quite good.
Should this piece be split into two questions in the FAQ?
- How to speed up NumPy code with `torch.compile`?
  - Use the fast cpu backend by enabling torch.compile
  - Run things on CUDA
- How to use NumPy code within your PyTorch code?
  - The wrap_numpy function

Reply:
I think splitting it is too much given how little information there is. I will add one example where we simply run things on CPU, implementing your first point "Use the fast cpu backend by enabling torch.compile"

Reply:
Nevermind, after writing it I chose to split it into two sections, although they are different to the ones you proposed.